app/components/metadata-form/index.js

```diff
@@ -1,47 +1,61 @@
 import Component from '@glimmer/component';
-import stripEmptyArrays from 'pass-ui/util/strip-empty-arrays';
 import { action } from '@ember/object';
+import { SurveyModel } from 'survey-js-ui';
+import { DefaultLightPanelless } from 'survey-core/themes';
 
 export default class MetadataForm extends Component {
+  survey = null;
+
   @action
   setupMetadataForm() {
-    const componentContext = this;
-    const originalForm = this.args.schema;
-    const newForm = JSON.parse(JSON.stringify(originalForm));
-    if (!originalForm.options) {
-      newForm.options = {};
-    }
+    const surveySchema = this.args.schema;
+    const surveyData = this.args.data;
 
-    // form ctrls
-    newForm.options.form = {
-      buttons: {
-        Back: {
-          title: 'Back',
-          styles: 'pull-left btn btn-outline-primary',
-          click() {
-            componentContext.args.previousForm(stripEmptyArrays(this.getValue()));
-          },
-        },
-        Abort: {
-          label: 'Cancel',
-          styles: 'pull-left btn btn-outline-danger ml-2',
-          click() {
-            componentContext.args.cancel();
-          },
-        },
-        Next: {
-          label: 'Next',
-          styles: 'pull-right btn btn-primary next',
-          click() {
-            componentContext.args.nextForm(stripEmptyArrays(this.getValue()));
-          },
-        },
+    const customCss = {
+      body: 'pt-4',
+      page: {
+        // Remove default survey.js page class
+        root: '',
+      },
+      navigation: {
```

Comment:
I think we can get rid of this in favor of:

`this.survey.showCompleteButton = false;`

Below.